src/sentry/models/monitor.py

```diff
@@ -18,11 +19,42 @@
     sane_repr,
 )
 
+SCHEDULE_FREQ_MAP = {
+    'yearly': rrule.YEARLY,
```

Comment:
im also totally happy to rename these to just be 'year' and 'month' (singular nouns)